src/torque-offset-estimator.cpp

```diff
@@ -92,7 +94,9 @@ namespace dynamicgraph
       /* --- COMMANDS ---------------------------------------------------------- */
       void TorqueOffsetEstimator::init(const std::string &urdfFile,
                                        const Eigen::Matrix4d& m_torso_X_imu_,
-                                       const double& gyro_epsilon_)
+                                       const double& gyro_epsilon_,
+                                       const int& ffIndex_,
+                                       const int& torsoIndex_)
       {
         try {
```

Comment:
I'll change this. However I still maintain that curly braces _should not_ be in new-lines. :no_good_man: